xla/tsl/mkl/graph.bzl

```diff
@@ -16,13 +16,13 @@ def onednn_graph_cc_library(srcs = [], hdrs = [], deps = [], **kwargs):
     native.cc_library(
         srcs = if_graph_api(srcs),
         hdrs = if_graph_api(hdrs),
-        deps = if_graph_api(deps),
+        deps = if_graph_api(deps) + mkl_deps(),
         **kwargs
     )
 
 def onednn_graph_cc_test(
         srcs = [],
-        deps = [],
+        deps = [] + mkl_deps(),
```

Comment:
Nit: I think adding `mkl_deps()` to the `if_true = deps` line looks a bit cleaner.